### PR TITLE
update docs

### LINKED ISSUE #21: Rover example project does not explain the configuration required
We should add the configuration (ports, etc.) to the README. This is what it appears to be in source:

### Default configuration (can be overridden somehow)
PanMotor = ServoMotorPort.S1
TiltMotor = ServoMotorPort.S2
LeftMotor = EncoderMotorPort.M4
RightMotor = EncoderMotorPort.M1

### Hard-coded
Led = DigitalPort.D3, Color.Green
Led = DigitalPort.D4, Color.Green
Led = DigitalPort.D0, Color.Red
Led = DigitalPort.D5, Color.Red
UltrasonicSensor = DigitalPort.D7
UltrasonicSensor = DigitalPort.D6
SoundSensor = AnaloguePort.A3

Also, how do all of these components connect physically?

### PR DESCRIPTION
**Pull request by colombod**
Closes #21

### CODE REVIEW

**Comment by m-roberts** [comment]
I used `libs/onnxruntime-b99eaa9.zip`, because it was built against a commit that is ahead of 1.5.2. We should try and stick to proper upstream releases; but this had some ARM stuff in it that we stood to benefit from.

Perhaps we should keep the commit hash as a documented reference?

File: src/Examples/PiTop.Interactive.Rover/readme.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+## Setting up your pi-top
+
+To run the app the configuration of the rover is
+
+*  right encoder motor on port `M1`
+*  left encoder motor on port `M2`
+*  tilt servo on port `S2`
+*  pan servo on port `S1`
+*  front green leds on ports `D3` and `D4`
+*  back red leds on port `D0` and ``D5`
```
Comment (m-roberts) [inline src/Examples/PiTop.Interactive.Rover/readme.md]:
```suggestion
*  back red leds on port `D0` and `D5`
```

File: src/Examples/PiTop.Interactive.Rover/readme.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
 
 This app will bootsrtap a rover app that embeds .NET Interactive. The kernels are exposed both as stdio and http interface.
```
Comment (m-roberts) [inline src/Examples/PiTop.Interactive.Rover/readme.md]:
```suggestion
This app will bootstrap a rover app that embeds .NET Interactive. The kernels are exposed both as stdio and http interface.
```

**Comment by m-roberts** [comment]
Also, an explanation of how all of these components connect physically would be very helpful!

**Comment by m-roberts** [comment]
Changes to `setup-device.sh` should be mirrored in `update-device.sh`:
* ONNX Runtime should be re-installed
* NuGet links should be updated

**Review by m-roberts — CHANGES_REQUESTED** [review]
See comments

File: src/Examples/PiTop.Interactive.Rover/readme.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
...
+*  tilt servo on port `S2`
+*  pan servo on port `S1`
+*  front green leds on ports `D3` and `D4`
+*  back red leds on port `D0` and ``D5`
+*  front ultrasound on port `D7`
+*  back ultrasound on port `D6`
+*  sound sensor on port `A3`
+*  camera on plate usb port
+
 ## Streaming Camera depedencies
```
Comment (m-roberts) [inline src/Examples/PiTop.Interactive.Rover/readme.md]:
```suggestion
## Streaming Camera dependencies
```

File: src/Examples/PiTop.Interactive.Rover/readme.md [hunk 4]
```diff
@@ -2,6 +2,21 @@
...
 
 The rover app uses a special camera that streams the video over http, this has a dependency on the  [mjpg-streamer](https://github.com/jacksonliam/mjpg-streamer) project. Follow the instruction to isntalle the tool before running the app.
```
Comment (m-roberts) [inline src/Examples/PiTop.Interactive.Rover/readme.md]:
```suggestion
The rover app uses a special camera that streams the video over HTTP. This has a dependency on the [mjpg-streamer](https://github.com/jacksonliam/mjpg-streamer) project. Follow the install instructions before running.
```

**Comment by m-roberts** [comment]
Also, `rover.` seems to need to be `roverBody.` in the rover README.

File: src/Examples/PiTop.Interactive.Rover/readme.md (hunk 4 above)
Comment (colombod) [reply src/Examples/PiTop.Interactive.Rover/readme.md]: done

**Comment by colombod** [comment]
> Also, an explanation of how all of these components connect physically would be very helpful!

Not sure what you mean by that.

**Comment by colombod** [comment]
> I used `libs/onnxruntime-b99eaa9.zip`, because it was built against a commit that is ahead of 1.5.2. We should try and stick to proper upstream releases; but this had some ARM stuff in it that we stood to benefit from.
> 
> Perhaps we should keep the commit hash as a documented reference?

All this should go as we should only provide these packages via nuget packages. I am adding a doc but not sure why  as that is more for your reference, please add any additional steps there as I have no recollection how you and Jorge built them

**Pull request merged**